definitions/npm/enzyme_v2.3.x/flow_v0.104.x-/test_enzyme-v2.3.js

```diff
@@ -30,7 +30,7 @@ const StatelessComponent = () => <div />;
 shallow(<div />).find(ClassComponent);
 shallow(<div />).find(StatelessComponent);
 shallow(<div />).find({ a: 1 });
-// $FlowExpectedError
+// $FlowExpectedError[incompatible-cast]
```

Comment:
Looks like you've used the wrong error code here
```suggestion
// $FlowExpectedError[incompatible-call]
```